synapse/config/emailconfig.py

```diff
@@ -177,9 +191,15 @@ def read_config(self, config, **kwargs):
             filepath = os.path.join(
                 self.email_template_dir, email_password_reset_template_success_html
             )
-            self.email_password_reset_template_success_html_content = self.read_file(
+            self.email_password_reset_template_success_html = self.read_file(
```

Comment:
I think the old name was better tbh... it reflects the fact that this is the content of the template file, not (as per the other `*_template_html` vars) the path to the template file.